robolectric/src/main/java/org/robolectric/ParameterizedRobolectricTestRunner.java

```diff
@@ -98,6 +102,16 @@ protected void validateConstructor(List<Throwable> errors) {
     }
 
     @Override
+    protected Statement methodBlock(FrameworkMethod method, Config config, AndroidManifest appManifest, SdkEnvironment sdkEnvironment) {
+      configureShadows(sdkEnvironment, config);
+
+      DeepCloner deepCloner = new DeepCloner(sdkEnvironment.getRobolectricClassLoader());
+      parameters = deepCloner.clone(parameters);
```

Comment:
Why does the classloader need to be deep-cloned? Is there any way to do this without introducing `xstream` as a dependency?